build.sbt

```diff
@@ -22,7 +22,7 @@ val magnoliaScala2Version = "1.1.2"
 val magnoliaScala3Version = "1.1.4"
 
 val algebirdVersion = "0.13.9"
-val avroVersion = Option(sys.props("avro.version")).getOrElse("1.11.0")
+val avroVersion = Option(sys.props("avro.version")).getOrElse("1.8.2")
```

Comment:
Keep latest version here

Reply:
When using the latest Avro - mangolify is fine. It will fail in users' app, because:
1. Scio enforces "1.8.2"
2. Even if we increase Scio version - [it will fail in Beam](https://github.com/spotify/scio/pull/4576)

Reply:
Now I got your point

Reply:
Yes, we should only use APIs which are present in all versions.
I'm wondering if we should probably do the opposite: develop with 1.8.2 and test with 1.11.0 to make there the API is still there

Reply:
Yes, I found how to do it. Please check!